plugins/woocommerce/client/admin/client/settings-payments/components/other-payment-gateways.tsx

```diff
@@ -53,28 +57,8 @@ const usePaymentGatewayData = () => {
 	}, [] );
 };
 
-// TODO implement gateway images.
-// const AdditionalGatewayImages = ( {
-// 	additionalGateways,
-// }: {
-// 	additionalGateways: PaymentGateway[];
-// } ) => (
-// 	<>
-// 		{ additionalGateways.map( ( gateway ) => (
-// 			<img
-// 				key={ gateway.id }
-// 				src={ gateway.image_72x72 }
-// 				alt={ gateway.title }
-// 				width="24"
-// 				height="24"
-// 				className="other-payment-methods__image"
-// 			/>
-// 		) ) }
-// 		{ _x( '& more.', 'More payment providers to discover', 'woocommerce' ) }
-// 	</>
-// );
-
 export const OtherPaymentGateways = () => {
+	const [ isShow, setIsShow ] = useState( false );
```

Comment:
Nitpick: `isShow` wasn't very clear to me at first, what do you think about `isVisible` instead?

Reply:
good point - thanks, changed to `isExpanded` so it's even more clear!